src/Oscardb.jl

```diff
@@ -0,0 +1,232 @@
+module Oscardb
+
+using ..Oscar
+using ..Oscar: load
+
+# for ca certificates
+using NetworkOptions
+
+using Mongoc
+
+using URIs
+
+const OSCAR_DB = "oscar"
+const OSCAR_DEV_DB = "oscar-dev"
+
+"""
+      Database
+
+Type for referencing a specific database (usually the `polyDB`)
+"""
+struct Database
+   mdb::Mongoc.Database
+end
+
+"""
+      Collection
```

Comment:
```suggestion
    Collection
```
Similar in all the other docstring: use four spaces for indentation.